### DISCUSSION TITLE
Supermaven stops working

### DISCUSSION THREAD

**Discussion by failable** [zed-industries/zed#12412] [Feature Requests]
I used supermaven for the last couple days before I switched to Copilot in this recent week. But when I switch back Supermaven today, it stops working and no completions is provided.

<img width="210" alt="image" src="https://github.com/zed-industries/zed/assets/5475262/1d0fea77-e314-4532-a6ce-06101b05f73d">

Not sure if there is an authentication issue and can not find a way to re-login. 

Any help will be appreciated, thanks!

**Comment by webdiego** [comment]
The same, it is not suggesting anything

**Comment by danny-su** [comment]
Supermaven is still initializing, but auto-complete is available.

**Zed:**<br>
<img width="272" alt="image" src="https://github.com/zed-industries/zed/assets/12178855/1a2cfead-4800-47de-b525-ce2d4f4c0e28"><br>
<img width="580" alt="image" src="https://github.com/zed-industries/zed/assets/12178855/cacb0305-9eac-4bc3-affc-488f6f6c7f0e">

<hr>

**RustRover:**<br>
<img width="218" alt="image" src="https://github.com/zed-industries/zed/assets/12178855/3f40d764-86c4-40d4-a766-aff231703ae1">

**Comment by ray-pH** [comment]
has anyone been able to get supermaven to work on linux?

**Comment by JosephTLyons** [comment]
I think @kevmo314 fixed this via:

https://github.com/zed-industries/zed/pull/14834

I'm going to close this out. Feel free to open new issues around Supermaven that you run into!
